config/init/Android.mk

```diff
@@ -43,6 +43,15 @@ LOCAL_MODULE_CLASS := ETC
 LOCAL_MODULE_PATH := $(TARGET_OUT_VENDOR_ETC)/init
 include $(BUILD_PREBUILT)
 
+include $(CLEAR_VARS)
+LOCAL_MODULE := init.exynos9820.root
+LOCAL_SRC_FILES := init.exynos9820.root.rc
+LOCAL_MODULE_TAGS := optional
+LOCAL_MODULE_STEM := init.exynos9820.root.rc
+LOCAL_MODULE_CLASS := ETC
+LOCAL_MODULE_PATH := $(TARGET_OUT_VENDOR_ETC)/init
```

Comment:
I would do it like we have it with yoshino, install it to `$(TARGET_OUT_VENDOR_ETC)/init/hw`.

Reply:
done